test/drbgtest.c

```diff
@@ -470,29 +474,36 @@ static int error_check(DRBG_SELFTEST_DATA *td)
 
     /* Test explicit reseed with entropy source failure */
     t.entropylen = 0;
-    if (!TEST_int_le(RAND_DRBG_reseed(drbg, td->adin, td->adinlen, 0), 0)
+    if (!TEST_false(RAND_DRBG_reseed(drbg, td->adin, td->adinlen, 0))
```

Comment:
+1: `TEST_false()` is _much_ better readable than `TEST_int_le()`.

Reply:
I added the check to a few more places in the function, where the RAND_DRBG_*() > 0 checks still remained.